meilisearch/tests/settings/get_settings.rs

```diff
@@ -113,7 +113,8 @@ async fn secrets_are_hidden_in_settings() {
                 "default": {
                     "source": "rest",
                     "url": "https://localhost:7777",
-                    "apiKey": "My super secret value you will never guess"
+                    "apiKey": "My super secret value you will never guess",
+                    "dimensions": 4,
```

Comment:
Added `dimensions` because the indexer makes an API call to estimate dimensions even if there is no document in the DB